PocketKit/Sources/PocketKit/MyList/SortFilterMenu/SortMenuModel.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+enum SortSection: String, Hashable, CaseIterable {
+    case sortBy = "Sort by"
+}
+
+enum SortRow: String, Hashable {
```

Comment:
Since this object actually represents more than just a row in some contexts (e.g. the value that is saved in user defaults) what do you think about renaming to something a bit more generic like `SortOption`?

Reply:
Sounds good